internal/oapiffi/ffi2swagger_test.go

```diff
@@ -64,6 +65,11 @@ func testFFI() *fftypes.FFI {
 					},
 				},
 			},
+			{
+				Name:     "method2",
+				Pathname: "method2",
+				/* no params */
```

Comment:
I _think_ even when there are no params, the FFI parser wants to at least see `params: []`

Reply:
I think this unit test confirms that `nil` and `&fftypes.FFIParams{}` are effectively the same thing here. Either way the JSON schema will get generated with an `input` object that contains no properties.